airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -86,7 +90,7 @@ def get_assets(
 
 @assets_router.get(
     "/events",
-    responses=create_openapi_http_exception_doc([404]),
+    responses=create_openapi_http_exception_doc([401, 403, 404]),
```

Comment:
We need to remove `401` and `403` those are automatically added now thanks to https://github.com/apache/airflow/pull/43932

Reply:
done